docs/Contributing/Seeding-Data.md

```diff
@@ -2,47 +2,80 @@
 
 When developing Fleet, it may be useful to create seed data that includes users and teams.
 
+
+> In order to run scripts that make use of premium features, make sure you have started the server with the correct flags as described in [Testing](./testing-and-local-development.md#license-key).
+
 Check out this Loom demo video that walks through creating teams seed data:
 https://www.loom.com/share/1c41a1540e8f41328a7a6cfc56ad0a01
 
-For a text-based walkthrough, check out the following steps:
+For a text-based walkthrough, follow these steps:
+
+## Create an environment variable file
 
 First, create an `env` file with the following contents:
 
 ```
 export SERVER_URL=https://localhost:8080 # your Fleet server URL and port
 export CURL_FLAGS='-k -s' # set insecure flag
-export TOKEN=eyJhbGciOi... # your login token
+export TOKEN=eyJhbGciOi... # your api token
 ```
 
-Next, set the `FLEET_ENV_PATH` to point to the `env` file. This will let the scripts in the `fleet/` folder source the env file.
+Conveniently, you can store the `env` file as `fleet_env` in the `./code/fleet-api-env` directory.
+
+## Apply your environment variable file
+
+Next, set the `FLEET_ENV_PATH` to point to the `env` file using the command line.
 
 ```
-export FLEET_ENV_PATH=/Users/victor/fleet_env
+export FLEET_ENV_PATH=./code/fleet-api-env/fleet_env
```

Comment:
Definitely good to change this from `Users/victor`, but I wonder if this is clear enough. What about `export FLEET_ENV_PATH=/path/to/fleet/fleet_env`?

Reply:
I realized why I did this, the code on this page in main refers to the tools/api directory using `./tools/api/fleet/teams/create_premium` so I was trying to make it cohesive with that.

Reply:
100% I just realized I had created that directory soooooo long ago on my laptop I just assumed it was a part of the fleet directory!